### PR TITLE
Adding Adobe Photoshop CS6

### PR DESCRIPTION
**Pull request by timothybasanov**
Installed using this script and verified success.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
All extraneous comments need to be removed.

Also, without a working `uninstall`, this won’t be merged.

**Comment by timothybasanov** [comment]
I used https://github.com/caskroom/homebrew-versions/blob/master/Casks/adobe-photoshop-cs5.rb as a reference.
Should that one be fixed as well?

**Comment by vitorgalvao** [comment]
It should. Actually, I’ll remove that one, as it was a candidate for removal in the last purge.

**Comment by timothybasanov** [comment]
Ok. Updated commit: 130698f327756ea705fa76ef0eb4779c522c3e82.

Now it supports both installation and uninstallation.
I get deployment xml from Adobe, replace "install" with "remove" and run against an installer.
This results in a clean deinstallation.

I do create a temporary file for a modified version of a script, unsure if this is how it's supposed to be done.

File: Casks/adobe-photoshop-cs6.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+require 'tempfile'
```
Comment (vitorgalvao) [inline Casks/adobe-photoshop-cs6.rb]: This needs to go. No other casks do this.

**Comment by vitorgalvao** [comment]
You didn’t run `brew cask audit` and `brew cask style --fix` on this before submission, hence why the build is failing.

**Comment by timothybasanov** [comment]
Done. Build is green. I removed 'require temple' and replaced it with `mktmp` call.

**Comment by vitorgalvao** [comment]
Pinging @winkelsdorf for a review.

**Comment by timothybasanov** [comment]
Any updates?

**Comment by vitorgalvao** [comment]
@winkelsdorf [is busy at the moment](https://github.com/caskroom/homebrew-cask/issues/17398#issuecomment-197048004). Since Adobe is very much user-hostile and usable casks for them are difficult to get right, I’d still like for him to take a look before merging, since he has done the most research on this.

**Comment by winkelsdorf** [comment]
@timothybasanov @vitorgalvao I now had the chance to fully test this cask. Creating an uninstall xml from the install xml was an excellent idea, Timothy! 

I would like to use a reference to the ```name``` variable instead of typing it multiple times. This needs to be improved in my previous casks later, too. Same for staged_path which you already implemented.

Furthermore I would put the uninstall xml in the same directory as the install xml, so we can reference it from the uninstall stanza without invoking everything via shell.

Please see my proposed changes above, I created a PR in your fork.

All tests ran fine.

If you agree to those changes, please squash the commits.

Cheers,
Frederik

**Comment by timothybasanov** [comment]
Done!

**Comment by winkelsdorf** [comment]
@timothybasanov Thank you!

@vitorgalvao Can be safely merged :)

**Comment by vitorgalvao** [comment]
Travis found a number of errors:

```
C: 24:  1: Trailing whitespace detected.
C: 25: 89: Surrounding space missing for operator |.
C: 25:101: Surrounding space missing for operator |.
C: 30: 22: Use %w or %W for an array of words.
C: 33: 23: Align the elements of a hash literal if they span more than one line.
```

`brew cask style --fix` should be able to fix a few of those.

**Comment by winkelsdorf** [comment]
My fault.

Actually ```brew cask style --fix``` is able to fix *all* of them :)

@timothybasanov Could you please run the above command against ```adobe-photoshop-cs6.rb```? Easier than having me doing it and creating a new PR for your own repo.

**Comment by timothybasanov** [comment]
Done!

**Comment by vitorgalvao** [comment]
Thank you both for your work on this.

**Pull request merged**